packages/ember-metal/lib/run_loop.js

```diff
@@ -625,15 +623,15 @@ run.debounce = function() {
     of the trailing edge of the wait interval. Defaults to true.
   @return {Array} Timer information for use in cancelling, see `run.cancel`.
 */
-run.throttle = function() {
+run.throttle = () => {
   return backburner.throttle.apply(backburner, arguments);
 };
 
 // Make sure it's not an autorun during testing
 function checkAutoRun() {
   if (!run.currentRunLoop) {
-    Ember.assert("You have turned on testing mode, which disabled the run-loop's autorun." +
-                 " You will need to wrap any code with asynchronous side-effects in a run", !Ember.testing);
+    Ember.assert(`You have turned on testing mode, which disabled the run-loop\'s autorun.
```

Comment:
This escaping is unnecessary (it doesn't do anything wrong, but it's not needed).